### PR TITLE
1.update delete garbage and punctuations tags;

### PR DESCRIPTION
**Pull request by wangyongqing0731**
2.rename files;
3. reorganize analyze_meta.py into specialized tools for kaldi, which is resolved for the issues 12

### CODE REVIEW

File: toolkits/kaldi/gigaspeech_data_prep.sh [hunk 1]
```diff
@@ -25,58 +25,68 @@ if [ $# -eq 4 ]; then
...
+garbage_tags="<SIL> <MUSIC> <NOISE> <OTHER>"
+punctuations_tags="<COMMA> <EXCLAMATIONPOINT> <PERIOD> <QUESTIONMARK>"
+
 declare -A subsets
 subsets=([train]="XL" [dev]="DEV" [test]="TEST")
 
-meta_dir=$data_dir/${prefix}corpus/meta
+corpus_dir=$data_dir/${prefix}corpus/
 if [ $stage -le 1 ]; then
+  echo "$0: Analyze meta into $corpus_dir"
```
Comment (chenguoguo) [inline toolkits/kaldi/gigaspeech_data_prep.sh:36]: Analyze --> Extract?

File: toolkits/kaldi/extract_corpus_for_kaldi.py [hunk 2]
```diff
@@ -40,14 +40,12 @@ def meta_analysis(input_json, output_dir, pipe):
            open(f'{output_dir}/text', 'w') as utt2text, \
```
Comment (chenguoguo) [inline toolkits/kaldi/extract_corpus_for_kaldi.py:40]: What about calling this file extract_meta.py?

**Review by chenguoguo — CHANGES_REQUESTED** [review]
Generally looks good to me but left two comments.

**Comment by dophist** [comment]
LGTM

**Pull request merged**